src/lib/components/ui/ArticleList.svelte

```diff
@@ -82,23 +94,38 @@
 		class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-y-4 md:gap-y-10 gap-x-6 justify-center"
 	>
 		{#each filteredArticles.slice(0, visibleArticles) as article, index}
-			{#if index === visibleArticles - ARTICLES_PER_PAGE}
-				<div id={`article-${index}`} bind:this={newArticleRef}>
-					<ArticleCard {article} />
-				</div>
-			{:else}
-				<div id={`article-${index}`}>
-					<ArticleCard {article} />
-				</div>
-			{/if}
+			<div transition:slide={{ delay: 100 * (index % ARTICLES_PER_PAGE) }}>
```

Comment:
why are you increasing the delay for each article?

Reply:
The delay is specifically for the slide animation effect.
For the loading state of the article to have a smoother "slide in" animation effect when the load more button is clicked